types/ui_schema/ui_options.ts

```diff
@@ -26,7 +26,7 @@ interface SignatureFieldUIOptions extends CommonCustomFieldUIOptions {
   'ui:x-backgroundColor'?: string
 }
 
-type ColumnNumber = 1 | 2 | 3 | 4
+export type ColumnNumber = 1 | 2 | 3 | 4
```

Comment:
这个名字感觉有点太泛了
1. 改下名字（或者可能加个 scope 之类的？）
2. 加个文档

```suggestion
/** The number of columns for options in radio button or check box fields. */
export type OptionColumns = 1 | 2 | 3 | 4
```
------
还有种处理方式是使用 `RadioButtonFieldUIOptions["ui:x-columns"]`，而不是直接用这个 `type` 呢？）

cc @moonRider

Reply:
第二种处理方式行不通，`RadioButtonFieldUiOptions` 是 `select` 和 `radio` 组成的联合，一个联合有这个类型，另一个没有，ts 没法推出来，而且 checkbox 那也在用，换个名字 `optionsColumns` 好点吧